packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -533,19 +615,12 @@ describe("toMapTree", () => {
 		});
 
 		it("List containing `undefined` (throws if fallback type is not allowed by the schema)", () => {
```

Comment:
given this description, it seems like this test should also cover the case where the fallback type is included, and it doesn't through (though thats interpreting the "if" as if and only if". Regardless that would be a good thing to test, either here or a almost idencal test next to it.

Reply:
The test above covers that case. I've updated the phrasing.